src/eth1spec/evm.py

```diff
@@ -9,13 +9,22 @@
 from dataclasses import dataclass
 from typing import List, Tuple
 
-from .eth_types import U256, Address, Hash32, Log, State
-from .number import Uint
+from .eth_types import EMPTY_ACCOUNT, U256, Address, Hash32, Log, State, Uint
 
-ADD = b"\x01"
-"""
-Opcode for the addition (`+`) instruction.
-"""
+ops = {
+    "ADD": b"\x01",
+    "PUSH1": b"\x60",
+    "SSTORE": b"\x55",
+}
+
+
+VERY_LOW = 3
+
+gas_schedule = {
```

Comment:
Should we make this an ENUM? It will just make the access pattern a bit easier to read I guess. Let me know your thoughts.

Reply:
Yeah, I think an enum could be good here. We can update it in a future PR.